test/corpus/statements.txt

```diff
@@ -671,102 +683,105 @@ async def d(a:str="default", b=c) -> None:
 
 (module
   (function_definition
-    name: (identifier)
-    parameters: (parameters)
-    body: (block
+    (identifier)
+    (parameters)
+    (block
       (expression_statement
         (identifier))))
   (function_definition
-    name: (identifier)
-    parameters: (parameters
+    (identifier)
+    (parameters
       (identifier))
-    body: (block
+    (block
       (expression_statement
         (identifier))))
   (function_definition
-    name: (identifier)
-    parameters: (parameters
+    (identifier)
+    (parameters
       (identifier)
       (identifier))
-    body: (block
+    (block
       (expression_statement
         (identifier))))
   (function_definition
-    name: (identifier)
-    parameters: (parameters
+    (identifier)
+    (parameters
       (typed_parameter
         (identifier)
-        type: (type
+        (type
           (identifier))))
-    body: (block
+    (block
       (expression_statement
         (identifier))))
   (function_definition
-    name: (identifier)
-    parameters: (parameters
+    (identifier)
+    (parameters
       (typed_parameter
         (identifier)
-        type: (type
+        (type
           (attribute
-            object: (identifier)
-            attribute: (identifier)))))
-    body: (block
+            (identifier)
+            (identifier)))))
+    (block
       (expression_statement
         (identifier))))
   (function_definition
-    name: (identifier)
-    parameters: (parameters
+    (identifier)
+    (parameters
       (typed_parameter
         (identifier)
-        type: (type
+        (type
           (subscript
-            value: (identifier)
-            subscript: (identifier)))))
-    return_type: (type
+            (identifier)
+            (identifier)))))
+    (type
       (identifier))
-    body: (block
+    (block
       (expression_statement
         (identifier))))
   (function_definition
-    name: (identifier)
-    parameters: (parameters
+    (identifier)
+    (parameters
       (identifier)
       (default_parameter
-        name: (identifier)
-        value: (identifier))
+        (identifier)
+        (identifier))
```

Comment:
It looks like the field names got removed from some of these expectations. Could you restore those?

Reply:
done